test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/XmlDataContractSerializerInputFormatterTest.cs

```diff
@@ -293,17 +292,14 @@ public void SetMaxDepth_ThrowsWhenMaxDepthIsBelowOne()
             Assert.True(context.HttpContext.Request.Body.CanRead);
         }
 
+        [ReplaceCulture]
```

Comment:
Do the encoding names used in the error message vary per-locale? If not, this isn't needed.